### DISCUSSION TITLE
Filter HTTP 500, 301 and 302 by urls

### DISCUSSION THREAD

**Discussion by jurgonaut** [allinurl/goaccess#2223] [General]
I'm playing around with goaccess and I'm considering using it to monitor my website, how every there are two statistics that I'm missing. These are the ability to see the urls that resulted in HTTP 500, 301 and 302istend of only the number of occurrences. Currently I'm testing the software out and I'm creating a html report with the command:
`goaccess some.log -a -o report.html`
Did I miss something? Is there a way to achieve what I want? In the future I plan to use goaccess in real time on a server so if a solution applies to only that scenario it's still good.

**Comment by allinurl** [comment]
To parse a specific status code, e.g., 500 (Internal Server Error):

    # awk '$9~/500/' access.log | goaccess - -o report.html

Or multiple status codes:

    # tail -f -n +0 access.log | awk '$9~/3[0-9]{2}|5[0-9]{2}/' | goaccess -o out.html - --real-time-html

Some examples can be found on the [**man page**](https://goaccess.io/man#examples).

#117 will address this much more elegantly. I'm working on this as we speak though.
